code/game/machinery/firealarm.dm

```diff
@@ -124,33 +123,6 @@
 							span_notice("You emag [src], disabling its thermal sensors."))
 	playsound(src, "sparks", 50, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
 
-/obj/machinery/firealarm/should_atmos_process(datum/gas_mixture/air, exposed_temperature)
-	return (exposed_temperature > T0C + 200 || exposed_temperature < BODYTEMP_COLD_DAMAGE_LIMIT) && !(obj_flags & EMAGGED) && !machine_stat
-
-/obj/machinery/firealarm/atmos_expose(datum/gas_mixture/air, exposed_temperature)
-	if(!detecting)
-		return
-	if(!triggered)
-		triggered = TRUE
-		myarea.triggered_firealarms += 1
-		update_appearance()
-	alarm()
```

Comment:
Did you get rid of the alarm sound?

Reply:
no this is for atmos_expose trigger, alarm sound is still there

Reply:
Could you make the alarm sound trigger from the fire alarm on atmos issues? or am I misunderstanding and that already happens

Reply:
That and maybe? the unique icon state, though that's maybe not needed anymore

Reply:
the alarm sound still play if you trigger it by hand, they don't have the atmos check on them anymore